pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "pdm.backend"
 
 [project]
 name = "algobattle-base"
-version = "4.0.0-rc1"
+version = "4.0.0rc2"

```

Comment:
Is the naming without the dash intended or required anywhere else? If not I would like to keep the formatting consistent.

Reply:
Python normalizes version numbers as described in [PEP 440](https://peps.python.org/pep-0440/#pre-release-separators), the spelling without the dash is the normalized form. I don't particularly mind either, my linter/autoformatter just prefers this one but we can revert it back if you prefer.

Reply:
I See, I was not aware of this PEP. Let us keep it your way then.